### PR TITLE
[4.0] upgrade: Skip powered off compute nodes during the upgrade prechecks

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

File: crowbar_framework/app/models/api/crowbar.rb [hunk 1]
```diff
@@ -91,7 +91,12 @@ def health_check
         unready = []
         # We are ignoring the ceph nodes, as they should already be in crowbar_upgrade state
         NodeObject.find("NOT roles:ceph-*").each do |node|
-          unready << node.name unless node.ready?
+          next if node.ready?
+          if node.state == "shutdown" && node.roles.include?("nova-compute-kvm")
```
Comment (skazi0) [inline crowbar_framework/app/models/api/crowbar.rb]: do we want this only for kvm?

Comment (jsuchome) [reply crowbar_framework/app/models/api/crowbar.rb]: I think we support only kvm compute nodes during the upgrade ... but as this is ECP specific anyway - are there any different ones?

Comment (skazi0) [reply crowbar_framework/app/models/api/crowbar.rb]: True, ECP only has KVM.

**Comment by jsuchome** [comment]
Closing in favor of https://github.com/crowbar/crowbar-core/pull/1749

**Pull request closed without merging**